src/components/Trade/TradeConfirm/TradeConfirm.tsx

```diff
@@ -141,10 +141,13 @@ export const TradeConfirm = ({ history }: RouterProps) => {
     gasFeeToTradeRatioPercentage > gasFeeToTradeRatioPercentageThreshold
 
   const txLink = useMemo(() => {
-    if (trade.sources[0].name === 'Osmosis') {
-      return `${osmosisAsset?.explorerTxLink}${txid}`
-    } else {
-      return `${trade.sellAsset?.explorerTxLink}${txid}`
+    switch (trade.sources[0].name) {
+      case 'Osmosis':
+        return `${osmosisAsset?.explorerTxLink}${txid}`
+      case 'CowSwap':
```

Comment:
Would prefer `Dex.CowSwap` but since we don't have `Dex.Osmosis`, that's fine for consistency